### PR TITLE
Add atari ppo example

### LINKED ISSUE #272: test_reward doesn't change in example
- [ ] I have marked all applicable categories:
    + [ ] exception-raising bug
    + [ ] RL algorithm bug
    + [ ] documentation request (i.e. "X is missing from the documentation.")
    + [ ] new feature request
- [x] I have visited the [source website](https://github.com/thu-ml/tianshou/)
- [x] I have searched through the [issue tracker](https://github.com/thu-ml/tianshou/issues) for duplicates
- [x] I have mentioned version numbers, operating system and environment, where applicable:
  ```python
  import tianshou, torch, sys
  print(tianshou.__version__, torch.__version__, sys.version, sys.platform)
  ```
I am using tianshou 0.3.0 post1. When I'm trying to run the pong_ppo.py in examples (python `pong_ppo.py`), in the 100 epochs, the test_reward never change. Thus, the pong cannot play in the final Collector.collect().

Am I mistaking something?

### LINKED ISSUE #374: Atari example for policy-based method like A2C, PPO
Is there any example using A2C, PPO to train atari ?

### LINKED ISSUE #440: Atari CNN Input Scale Issue
Hello,

It seems that the input of the CNN network does not be normalized by dividing 255 in the given examples. This is quite different from several public implementations. Any explanation for this?

Thanks,
Ziniu

### LINKED ISSUE #497: PPO example for Spaceinvaders
May I ask if there are any examples using PPO to train spaceinvaders ?

I noticed there was a pong_ppo.py file in the past opened issue. Is it going to achieve good performance on spaceinvaders simply by changing the env_id?

### PR DESCRIPTION
**Pull request by nuance1979**
- [x] I have marked all applicable categories:
    + [ ] exception-raising fix
    + [ ] algorithm implementation fix
    + [ ] documentation modification
    + [x] new feature
- [x] I have reformatted the code using `make format` (**required**)
- [x] I have checked the code using `make commit-checks` (**required**)
- [x] If applicable, I have mentioned the relevant/related issue(s)
- [x] If applicable, I have listed every items in this Pull Request below

I needed a policy gradient baseline myself and it has been requested several times (#497, #374, #440). I used https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ppo_atari.py as a reference for hyper-parameters.

Note that using lr=2.5e-4 will result in "Invalid Value" error for 2 games. The fix is to reduce the learning rate. That's why I set the default lr to 1e-4. See discussion in https://github.com/DLR-RM/rl-baselines3-zoo/issues/156.

### CODE REVIEW

**Comment by Trinkle23897** [comment]
Things to check:

1. obs/255
2. shared policy-value network till which MLP layer

**Comment by nuance1979** [comment]
> Things to check:
> 
> 1. obs/255

I did some experiments (see below) and it seemed that normalizing obs did not help. Since our previous exps do not normalize obs, I'd suggest keeping it as an option ("--scale-obs") but turning it off by default.

> 2. shared policy-value network till which MLP layer

I made the change so that the network structure matches other (sb3, cleanrl) implementations, i.e., sharing NatureCNN + 1 Linear-512, then both actor and critic are just one linear layer. The results seems to be about the same as my old structure. I'll rerun the exps and update the plots.

- With sb3 network structure: best_reward=1240
SpaceInvaders_rew linear_actor_critic

- With sb3 network structure + obs/255: best_reward=888
SpaceInvaders_rew linear_actor_critic scale_obs

- With my old network structure: best_reward=1184
SpaceInvaders_rew

- With my old network structure + obs/255: best_reward=787
SpaceInvaders_rew scale_obs

**Review by Trinkle23897 — APPROVED** [review]
